core/arch/arm/plat-stm32mp1/scmi_server.c

```diff
@@ -1025,7 +1025,7 @@ int32_t plat_scmi_perf_level_latency(unsigned int channel_id,
 	if (!perfd)
 		return SCMI_NOT_FOUND;
 
-	/* Measured latency on STM32MP13 is around 650uS */
+	/* Measured latency on STM32MP13 is around 650uS so set 1mS */
```

Comment:
This inline comment should rather go in **plat-stm32mp1/conf.mk** where `CFG_STM32MP_OPP_LATENCY_US` default value is defined.`